go.mod

```diff
@@ -26,6 +26,7 @@ require (
 	k8s.io/api v0.19.4
 	k8s.io/apimachinery v0.19.4
 	k8s.io/client-go v0.19.4
+	sigs.k8s.io/controller-runtime v0.7.0 // indirect
```

Comment:
Something is still wrong and 0.7.0 won't compile.. Can you try deleting this line and run `go mod tidy`?

Reply:
Done and pushed